.github/workflows/ci.yml

```diff
@@ -204,6 +204,7 @@ jobs:
         command: pnpm install --frozen-lockfile
     - name: Run composer package test suite
       run: cd packages/composer && pnpm test
+      if: ${{ matrix.os != 'windows-latest' && matrix.node-version != 20 }}
```

Comment:
please exclude it in the matrix above https://github.com/platformatic/platformatic/pull/1305/files#diff-b803fcb7f17ed9235f1e5cb1fcd2f5d3b2838429d4368ae4c57ce4436577f03fR190